setup.py

```diff
@@ -48,6 +48,42 @@
 It should work without additional code in any environment with
 Python >= 2.4, ctypes and an pre-built usb backend library
 (currently, libusb 0.1.x, libusb 1.x, and OpenUSB).
-"""
+""",
+    classifiers=[
+        'Development Status :: 4 - Beta',
```

Comment:
Seems that beta is the best fit here, as there is no frozen/rc status in trove.

Regarding the duplicate information, I was just predicting me updating one file and forgetting to update the other.